frontend/ui/src/components/kanban-board/task-card.tsx

```
@@ -1,114 +1,293 @@
+              <div className="gap-2 space-y-2 py-2">
+                <div className="flex w-full items-center justify-between gap-1 uppercase text-zinc-500">
+                  <span>{subtask.name}</span>
+                  <span>{subtask?.progress}%</span>
+                </div>
+                <div
+                  className="relative flex h-2 w-full items-center gap-1 rounded-full bg-gray-200 before:absolute before:left-0 before:top-0 before:h-2 before:w-[var(--progress)] before:rounded-full before:bg-green-400"
+                  style={{ '--progress': `${subtask?.progress}%` } as React.CSSProperties}
+                />
+              </div>
+            </>
+          ))}
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)